city_furniture/views/wfs.py

```diff
@@ -58,35 +78,87 @@ class CityFurnitureWFSView(WFSView):
     }
 
     furniture_signpost_base_fields = [
-        "id",
-        "source_name",
-        "source_id",
-        "project_id",
-        field("owner_name_fi", model_attribute="owner.name_fi"),
-        field("responsible_entity_name", model_attribute="responsible_entity.name"),
-        "location",
-        "location_name",
-        "location_additional_info",
-        "direction",
-        field("device_type_code", model_attribute="device_type.code"),
-        field("color_code", model_attribute="color.rgb"),
-        "height",
-        field("mount_type_description_fi", model_attribute="mount_type.description_fi"),
-        field("parent_id", model_attribute="parent.id"),
-        "order",
-        "pictogram",
-        "value",
-        "size",
-        "arrow_direction",
-        field("target_name_fi", model_attribute="target.name_fi"),
-        "content_responsible_entity",
-        "text_content_fi",
-        "text_content_sw",
-        "text_content_en",
-        "validity_period_end",
-        "validity_period_start",
-        "additional_material_url",
-        "lifecycle",
+        DescribedFeatureField("id", description=_("ID of the Furniture Signpost.")),
```

Comment:
Are there translations somewhere for these strings? Or will we add them later?

Reply:
Yep, you're right, they are only intended to be in English. Removed `gettext_lazy` functions from description strings.